vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -271,23 +198,27 @@ private void exportVariants(VariantContextWriter writer) {
     private VCFHeader getOutputVcfHeader() {
         // get VCF header(s) and write them to output file(s)
         logger.info("Generating VCF header ...");
-        List<VariantSource> sources = exporter.getSources(variantSourceDBAdaptor, studies, files);
+        List<VariantSource> sources = exporter.getSources(variantSourceService, studies, files);
         VCFHeader header = null;
         try {
-            boolean excludeAnnotations = ((String) query.get("exclude")).contains("annotation");
+            List<String> exclusions = query.getExclusions();
+            boolean excludeAnnotations = true;
```

Comment:
I find this a little bit confusing. The annotations are being excluded by default, but, if there is an exclude parameter not containing "annotation", they are being included. So, the only way to include the annotation is to use an exclude parameter (with no "annotation").

Is like there are two "default" values: exclude annotations if there are no exclusions in the query, and include annotations if there are exclusions in the query. If this is intended, maybe one use case is the CLI and the other the WS. Or for different WS endpoints. If this is intended, I suggest changing the API to make it more clear. If this is not intended, then there is a bug here and it needs to be fixed.